__tests__/reducers/inputs.test.js

```diff
@@ -344,6 +348,23 @@ describe('setBaseURL', () => {
   })
 })
 
+describe('showResourceURIMessage', () => {
+  it('displays the Resource URI', () => {
+    initialState.editor.resourceURIMessage = {
+      show: false,
+      uri: '',
+    }
+
+    const result = showResourceURIMessage(initialState, {
+      type: 'SHOW_RESOURCE_URI_MESSAGE',
+      payload: 'http://example.com/foo/123',
+    })
+
+    expect(result.editor.resourceURIMessage.show).toBe(true)
+    expect(result.editor.resourceURIMessage.uri).toEqual('http://example.com/foo/123')
+  })
+})
```

Comment:
👍 .   question - how does editor.resourceURIMessage.show become false again?

Reply:
my read of the code is that that changing back isn't expected or implemented behavior for now.  i don't see anything to transform the state back in that direction in this PR.